tests/src/test/scala/common/rest/WskRest.scala

```diff
@@ -1294,21 +1313,35 @@ class RunWskRestCmd() extends FlatSpec with RunWskCmd with Matchers with ScalaFu
 
   def convertMapIntoKeyValue(params: Map[String, JsValue],
                              feed: Option[String] = None,
-                             web: Option[String] = None): JsArray = {
+                             web: Option[String] = None,
+                             oldParams: Vector[JsObject] = Vector[JsObject]()): JsArray = {
     var paramsList = Vector[JsObject]()
     params foreach { case (key, value) => paramsList :+= JsObject("key" -> key.toJson, "value" -> value.toJson) }
     paramsList = feed map { f =>
       paramsList :+ JsObject("key" -> "feed".toJson, "value" -> f.toJson)
     } getOrElse paramsList
+
+    if (paramsList.size == 0) {
+      paramsList = oldParams
+    }
+
     paramsList = web match {
       case Some("true") =>
         paramsList :+ JsObject("key" -> "web-export".toJson, "value" -> true.toJson) :+ JsObject(
           "key" -> "raw-http".toJson,
           "value" -> false.toJson) :+ JsObject("key" -> "final".toJson, "value" -> true.toJson)
+      case Some("yes") =>
+        paramsList :+ JsObject("key" -> "web-export".toJson, "value" -> true.toJson) :+ JsObject(
+          "key" -> "raw-http".toJson,
+          "value" -> false.toJson) :+ JsObject("key" -> "final".toJson, "value" -> true.toJson)
```

Comment:
Again, remove mutability and duplication. You can do this by have a `Seq` returned here, which you can add (or not add) to a list by using `++` like I did above.

To make this return a sequence (and remove duplication)

```scala
val webOpt = web.map {
  case "true" | "yes" => 
    Seq(
      JsObject("key" -> "web-export".toJson, "value" -> true.toJson),
      JsObject("key" -> "raw-http".toJson, "value" -> false.toJson),
      JsObject("key" -> "final".toJson, "value" -> true.toJson))
  case "false" | "no" =>
    Seq(
      ...
}
```

Makes sense?